src/components/Map/PlantingSiteMap.tsx

```diff
@@ -91,74 +90,92 @@ export default function PlantingSiteMap(props: PlantingSiteMapProps): JSX.Elemen
+          selectFillColor: getRgbaFromHex(theme.palette.TwClrBasePink300 as string, 0.6),
+          highlightFillColor: getRgbaFromHex(theme.palette.TwClrBasePink300 as string, 0.6),
+          lineColor: theme.palette.TwClrBasePink300 as string,
+          lineWidth: 2,
+        };
+      } else {
+        // temporary plot
+        return {
+          fillColor: getRgbaFromHex(theme.palette.TwClrBaseYellow300 as string, 0.2),
+          hoverFillColor: getRgbaFromHex(theme.palette.TwClrBaseYellow300 as string, 0.4),
+          selectFillColor: getRgbaFromHex(theme.palette.TwClrBaseYellow300 as string, 0.6),
+          highlightFillColor: getRgbaFromHex(theme.palette.TwClrBaseYellow300 as string, 0.6),
+          lineColor: theme.palette.TwClrBaseYellow300 as string,
+          lineWidth: 2,
+        };
       }
     },
-    [theme.palette.TwClrBaseGreen300, theme.palette.TwClrBaseLightGreen300, theme.palette.TwClrBaseBlue300]
-  );
-
-  const extractPlantingSite = useCallback(
-    (site: PlantingSite): MapSource => {
-      const renderAttributes = getRenderAttributes('site');
-
-      return {
-        ...MapService.extractPlantingSite(site),
-        ...renderAttributes,
-      };
-    },
-    [getRenderAttributes]
-  );
-
-  const extractPlantingZones = useCallback(
-    (site: PlantingSite): MapSource => {
-      const renderAttributes = getRenderAttributes('zone');
-
-      return {
-        ...MapService.extractPlantingZones(site),
-        ...renderAttributes,
-      };
-    },
-    [getRenderAttributes]
-  );
-
-  const extractSubzones = useCallback(
-    (site: PlantingSite): MapSource => {
-      const renderAttributes = getRenderAttributes('subzone');
-
-      return {
-        ...MapService.extractSubzones(site),
-        isInteractive: true,
-        annotation: {
-          textField: 'fullName',
-          textColor: theme.palette.TwClrBaseWhite as string,
-          textSize: 16,
-        },
-        ...renderAttributes,
-      };
-    },
-    [getRenderAttributes, theme.palette.TwClrBaseWhite]
+    [
+      theme.palette.TwClrBaseGreen300,
+      theme.palette.TwClrBaseLightGreen300,
+      theme.palette.TwClrBaseBlue300,
+      theme.palette.TwClrBasePink300,
+      theme.palette.TwClrBaseYellow300,
+    ]
   );
 
   // fetch polygons and boundaries
   useEffect(() => {
     const fetchPlantingSite = () => {
-      const site = extractPlantingSite(plantingSite);
-      const zones = extractPlantingZones(plantingSite);
-      const subzones = extractSubzones(plantingSite);
-
-      const includedSources = new Array<MapSource>();
-      if (layers === undefined || layers?.includes('Sub-Zones')) {
-        includedSources.push(subzones);
+      const sources = new Array<MapSource>();
+      if (mapData.temporaryPlot && (layers === undefined || layers?.includes('Monitoring Plots'))) {
+        sources.push({
+          ...mapData.temporaryPlot,
+          isInteractive: true,
```

Comment:
Just calling this out to put it on your radar.
Fwiw, isInteractive tells the Map component which ones to allow click/popups for.
For example, if all layers are off except the planting site, we want the planting site to be clickable and show a popup with plants stats, etc.
I guess dynamically set isInteractive based on what's in the layers. Figma has these various use-cases highlighted.
We can add these features later for sure, just wanted to call it out.

Reply:
Cool, I will add those things as I start to work through the interactivity with those map elements 👍